internal/fs/inode/dir.go

```diff
@@ -757,6 +769,9 @@ func (d *dirInode) readObjects(
 			cores[dirName] = implicitDir
 		}
 	}
+	if len(unsupportedPrefixes) > 0 {
+		logger.Errorf("Encountered unsupported prefixes during listing: %v", unsupportedPrefixes)
```

Comment:
I think, warning log would be better. As we are not sure, if this will generate error everytime.

Reply:
done